### PR TITLE
Release 3.4.0

### PR DESCRIPTION
**Pull request by mariushoch**

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
You cannot add new functionality in patch releases. Minor release is required.

http://semver.org

**Comment by JeroenDeDauw** [comment]
You cannot add new functionality in patch releases. A minor release is required. See http://semver.org

(I remember writing this before... is there another PR like this one somewhere?)

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Wikibase DataModel Services release notes
 
+## Version 3.3.1 (2016-02-20)
```
Comment (thiemowmde) [inline RELEASE-NOTES.md]: Must be 3.4.0.

**Comment by thiemowmde** [comment]
Erm, what happened here? https://github.com/wmde/WikibaseDataModelServices/compare/3.2.0...3.2.1 This is not a bugfix. What got tagged there? Who reviewed and merged this? What made this violation of quite some processes, rules and best practices necessary?

**Comment by JeroenDeDauw** [comment]
Oh, that's not good... same questions. Why not just have a new minor release as usual? Now semver compliance got messed up :(

**Comment by Benestar** [comment]
Please update this pr to release a 3.4 version

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Wikibase DataModel Services release notes
 
+## Version 3.4.0 (2016-02-22)
+* Add RestrictedEntityLookup::reset
```
Comment (Benestar) [inline RELEASE-NOTES.md]: Missing new line after heading

**Comment by Benestar** [comment]
I'm ok with releasing this today and there hasn't changed much since the last release anyways (https://github.com/wmde/WikibaseDataModelServices/compare/3.3.0...master).

Should we include the 3.2.x releases in the release notes?

**Comment by thiemowmde** [comment]
+2. I would not mention 3.2.1 and 3.2.2. I would even delete the releases here on GitHub and only keep the tags.

File: RELEASE-NOTES.md (hunk 2 above)
Comment (thiemowmde) [inline RELEASE-NOTES.md]: Should be:
- Added `RestrictedEntityLookup::reset`

Comment (Benestar) [inline RELEASE-NOTES.md]: `* Added `RestrictedEntityLookup::reset``

**Comment by mariushoch** [comment]
Amended the release notes

**Pull request merged**